app/controllers/spree/admin/orders_controller_decorator.rb

```diff
@@ -74,7 +74,7 @@ def index
   # This uses a new template. See mailers/spree/order_mailer_decorator.rb.
   def resend
     Spree::OrderMailer.confirm_email_for_customer(@order.id, true).deliver
-    flash[:success] = t(:order_email_resent)
+    flash[:success] = t('admin.orders.order_email_resent')
```

Comment:
This one is covered by Spree. I have the exact same change in https://github.com/Em-AK/openfoodnetwork/pull/9 as part of the soree upgrade. I'll close it as soon as this gets merged.